coalib/bears/Bear.py

```diff
@@ -38,10 +38,6 @@ class Bear(Printer, LogPrinterMixin):
     err(), you are expected to also terminate the bear run-through
     immediately.
 
```

Comment:
also, please end commit body with a period.

aaand "Deleted" -> "Deletes" as its the commit that does, not you